helpdesk-message-proxy-service/src/main/java/org/symphonyoss/symphony/bots/helpdesk/messageproxy/service/TicketService.java

```diff
@@ -157,4 +164,27 @@ public void sendIdleMessageToAgentStreamId(SymMessage message) {
       LOGGER.error("Could not send ticket message to agent stream ID: ", e);
     }
   }
+
+  public void sendMessageWhenRoomCreationFails(SymMessage symMessage) {
+    symMessage.setMessage(parseTemplate(SERVICE_ROOM_WAS_NOT_CREATED));
+
+    try {
+      symphonyClient.getMessagesClient().sendMessage(symMessage.getStream(), symMessage);
+    } catch (MessagesException e) {
+      LOGGER.error("Could not send message to client stream ID: ", e);
+    }
+  }
+
+  private String parseTemplate(String templateFilename) {
+    StringBuilder message = new StringBuilder();
+    InputStream resource = getClass().getClassLoader().getResourceAsStream(templateFilename);
```

Comment:
I don't think you require an XML file to define this message. You can create a constant field in the class.

Reply:
Done.